util/has_lib.js

```diff
@@ -69,13 +69,13 @@ function hasLdconfig () {
  */
 function hasFreetype () {
   try {
-    if (childProcess.execSync('pkg-config cairo --cflags-only-I 2>/dev/null | grep freetype2').length) {
+    if (childProcess.execSync('pkg-config cairo --cflags-only-I 2>/dev/null | ' +
+                              'grep freetype2').length) {
       return true
     }
   } catch (err) {
-    // noop
+    return false
   }
-  return false
```

Comment:
This change makes it so not all code paths return a value. I'm not positive, but it might be sufficient to do what's done in `hasPkgconfigLib`: just `try { execSync(...); return true; } catch (err) { return false }`.